backend/src/main/models/Assigned.ts

```diff
@@ -0,0 +1,23 @@
+'use strict';
+import {
+  Table,
+  Column,
+  Model,
+  PrimaryKey,
+  AutoIncrement,
+  ForeignKey,
+  BelongsTo,
+} from 'sequelize-typescript';
+import { Account } from './Account';
+import { Task } from './Task';
+
+@Table({ timestamps: true })
+export class Assigned extends Model {
+  @ForeignKey(() => Task)
```

Comment:
similar comment to the migration. I think you need to specify that it is a primary key for both.